lua/astrocommunity/ai/opencode-nvim/README.md

```diff
@@ -0,0 +1,7 @@
+# NickvanDyke/opencode.nvim
+
+Integrate the opencode AI assistant with Neovim — streamline editor-aware research, reviews, and requests.
+
+**Repository**: <https://github.com/NickvanDyke/opencode.nvim>
+
+Note: Installation of [OpenCode](https://opencode.ai/) is required. To access more models you'll need to auth with `opencode auth login`
```

Comment:
End-users should read the docs anyways. 
```suggestion
```

Reply:
you're not wrong, I just like trying to prevent foot guns. without a doubt people will come on disc asking question ahaha